src/__tests__/field/email.test.js

```diff
@@ -0,0 +1,27 @@
+import { isEmail, emailDomain } from '../../field/email';
+
+describe('field/email', () => {
+  describe('isEmail', () => {
+    it('validates correctly', () => {
+      expect(isEmail('test@test.com')).toBe(true);
+      expect(isEmail('test@test.com.br')).toBe(true);
+      expect(isEmail('test@移动.移动')).toBe(true);
+      expect(isEmail(1)).toBe(false);
+      expect(isEmail('test@testcom')).toBe(false);
+      expect(isEmail('test.test.com')).toBe(false);
+    });
+    it('returns false when there is a white space', () => {
+      expect(isEmail('test@test.com ')).toBe(false);
```

Comment:
This is a valid scenario if trim is enabled. However, a space in between should be invalid.